.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -924,7 +920,47 @@ stages:
           displayName: Publish release artifacts
           artifact: $(tracer_version)-release-artifacts
 
-        # We don't include the MSI artifacts as they're not signed
+        # We don't include the MSIs in the artifact upload as they're
+        # not signed, but we do include them in the push to Azure
+        - task: DownloadPipelineArtifact@2
+          displayName: Download x64 MSI
+          inputs:
+            artifact: windows-msi-x64
+            path: $(Build.ArtifactStagingDirectory)
+
+        - task: DownloadPipelineArtifact@2
+          displayName: Download x86 MSI
+          inputs:
+            artifact: windows-msi-x86
+            path: $(Build.ArtifactStagingDirectory)
+
+        - bash: |
+            az storage blob upload-batch \
+              --destination 'apm-dotnet-ci-artifiacts-master' \
```

Comment:
Looks like there's a typo in the storage blob name:

```suggestion
              --destination 'apm-dotnet-ci-artifacts-master' \
```

Reply:
If only I hadn't misspelled it in Azure too 🤦‍♂️ 
![image](https://user-images.githubusercontent.com/18755388/128829194-54bfde15-bb64-489b-9ce5-74aa51c2be68.png)

Reply:
Instead of hard-coding the value (with or without typos) I moved it to a variable instead, like the other Azure variables as it feels like they shouldn't be hard coded here